test/Grpc.AspNetCore.Server.ClientFactory.Tests/GrpcClientServiceExtensionsTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using Greet;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using NUnit.Framework;
+
+namespace Grpc.AspNetCore.Server.ClientFactory.Tests
+{
+    [TestFixture]
```

Comment:
nit: Why is this attribute needed? I don't see any setup code. Not a big deal since we do this everywhere though.

Reply:
It is required by nunit on test classes

Reply:
Should be optional if there are no setup code. I tried removing it on some tests and the tests still run.

Reply:
https://nunit.org/docs/2.6/testFixture.html

> Beginning with NUnit 2.5, the TestFixture attribute is optional for non-parameterized, non-generic fixtures. So long as the class contains at least one method marked with the Test, TestCase or TestCaseSource attribute, it will be treated as a test fixture.

Hmm. I'm going to be old school and say we should keep using it everywhere.